src/triage/component/postmodeling/contrast/group_evaluator.py

```diff
@@ -0,0 +1,325 @@
+
+    A model_group_id list is needed to instate the class.
+    '''
+    def __init__(self, model_group_id):
+        self.model_group_id = model_group_id
+
+        # Retrive model_id metadata from the model_metadata schema
+        model_metadata = pd.read_sql("""
+        WITH
+        individual_model_ids_metadata AS(
+        SELECT m.model_id,
+               m.model_group_id,
+               m.hyperparameters,
+               m.model_hash,
+               m.experiment_hash,
+               m.train_end_time,
+               m.train_matrix_uuid,
+               m.training_label_timespan,
+               mg.model_type,
+               mg.model_config
+            FROM model_metadata.models m
+            JOIN model_metadata.model_groups mg
+            USING (model_group_id)
+            WHERE model_group_id IN {}
+        ),
+        individual_model_id_matrices AS(
+        SELECT DISTINCT ON (matrix_uuid)
+               model_id,
+               matrix_uuid
+            FROM test_results.predictions
+            WHERE model_id = ANY(
+                SELECT model_id
+                FROM individual_model_ids_metadata
+            )
+        )
+        SELECT metadata.model_id,
+               metadata.model_group_id,
+               metadata.model_hash,
+               metadata.hyperparameters,
+               metadata.train_end_time,
+               metadata.train_matrix_uuid,
+               metadata.training_label_timespan,
+               metadata.model_type,
+               metadata.model_config,
+               test.matrix_uuid AS test_matrix_uuid
+        FROM individual_model_ids_metadata AS metadata
+        LEFT JOIN individual_model_id_matrices AS test
+        USING(model_id);
+        """.format(self.model_group_id),
+                   con=db_engine).to_dict('list')
+
+        # Add metadata attributes to model
+        self.model_id = model_metadata['model_id']
+        self.model_type = model_metadata['model_type']
+        self.train_end_time = model_metadata['train_end_time']
+        self.hyperparameters = model_metadata['hyperparameters']
+        self.model_hash = model_metadata['model_hash']
+        self.train_matrix_uuid = model_metadata['train_matrix_uuid']
+        self.test_matrix_uuid = model_metadata['test_matrix_uuid']
+        self.train_matrix = None
+        self.pred_matrix = None
+        self.preds = None
+        self.feature_importances = None
+        self.model_metrics = None
+
+    def __str__(self):
+        s = 'Model collection object for model_ids: {}'.format(self.model_id)
+        s += '\n Model Groups: {}'.format(self.model_group_id)
+        s += '\n Model types: {}'.format(self.model_type)
+        s += '\n Model hyperparameters: {}'.format(self.hyperparameters)
+        s += '\Matrix hashes (train,test): {}'.format((self.train_matrix_uuid,
+                                                      self.test_matrix_uuid))
+        return s
```

Comment:
Does `triage` runs its test in Python 3.6? The f-strings are only added until the 3.6 version. I don't want to mess up stuff.

Reply:
As we discussed, it's currently tested against v3.6 (see tox.ini) and its virtual environment is intended to be created with v3.6 (see .python-version). Beyond that, I _think_ we're already otherwise using v3.6 features ... though, I could be wrong.

Regardless, good thinking!